### PR TITLE
Feat/get jobs by crow fly distance

### PR DESCRIPTION
**Pull request by kepkap**
The issue is https://github.com/openmarketplaceengine/openmarketplaceengine/issues/112

The implementation 
get the last location of worker (we have it in redis)
use that location to query jobs from postgres by lat/lon range. In postgres we store lat/lon as floats. 
the query uses math formula to produce distance range from pick-up location to requested from-location

This in fact returns a crow-fly distances.

Actually this approach proved to be very sufficient for MVP, but weak points are of cause sql range queries over not indexed lon/lat.

### CODE REVIEW

File: api/job/v1beta1/job.proto [hunk 1]
```diff
@@ -53,4 +54,27 @@ message ExportJobItem {
 
 message ExportJobResponse {
   repeated ExportJobItem jobs = 1;
+}
+
+
+message GetAvailableJobsRequest {
+  string area_key = 1;
```
Comment (kevinmichaelchen) [inline api/job/v1beta1/job.proto:61]: what's an area key? let's add a comment for this field

File: api/job/v1beta1/job.proto [hunk 2]
```diff
@@ -53,4 +54,27 @@ message ExportJobItem {
...
+  string worker_id = 2;
+  double lange_limit = 3;
```
Comment (kevinmichaelchen) [inline api/job/v1beta1/job.proto:63]: typo? range_limit?

Comment (kepkap) [reply api/job/v1beta1/job.proto:63]: yep, thank for catching.

File: dom/job/job.go [hunk 3]
```diff
@@ -29,6 +30,23 @@ type Job struct {
...
+	FromLat float64
+	FromLon float64
+	Unit    DistanceUnit
+	Range   float64
+}
+
+type DistanceUnit string
+
+const Km DistanceUnit = "km"
+const Mile DistanceUnit = "mile"
```
[the remark this answers is not in the document]
Comment (kepkap) [reply dom/job/job.go:48]: I thgink i will add meters and make it default, but will not delete km and miles. maybe some long-range queries will benefit from distance units. 
If not used - we will delete km and miles later.

**Review by vlkolmakov — APPROVED** [review]

File: svc/job/job_test.go [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+	ctx := context.Background()
+
+	err = job.DeleteAll(ctx)
+	require.NoError(t, err)
+
+	fromLat := 40.633650
+	fromLon := -74.143650
+
+	job1 := &job.Job{
+		ID:        uuid.NewString(),
```
[the remark this answers is not in the document]
Comment (kepkap) [reply svc/job/job_test.go:35]: we have uuid 44 usages. i will not get rid of this in current PR, but  i will start using NewXid() and remove uuid dependency eventually. Thanks!

**Pull request merged**